Component/Text-Utilies/text-utilities.css

```diff
@@ -0,0 +1,67 @@
+@import url(/CSS/docs.css);
+
+html {
+    font-size: 62.5%;
+  }
+ 
+body {
+    font-family: 'Mate', serif;
+}
+
+
+.main-content-display{
+    display: flex;
+    flex-direction: column;
+    gap:1rem;
+}
+.heading-1,.heading-2,hr{
+    color:var(--primary)
+}
+
+.grey-text{
+    color:var(--grey-text);
+}
+
+.center-text{
+    text-align: center;
+}
+
+p{
+    text-align: justify;
+}
+
+li{
+    list-style: none;
+    margin:1rem 0;
+    font-size:1.655rem;
+}
+
+a {
+    text-decoration: none;
+    cursor: pointer;
+    color: inherit;
+  }
```

Comment:
Improvement: Don't apply styles on direct element name, instead use selectors(class). Or you can make default or global css file for repeating properties. Reason is that if make any new element by default this style will apply.

Reply:
I wanted these to get applied to all the components and the user who will use the component. I also mentioned them at the end of the text utilities. Still, I plan to change it at the later stages.

Reply:
Ya then it's fine. I also did this and In yesterday's PR review session MAs told that dont do this that's why I told you.